pykern/pkcli/github.py

```diff
@@ -531,7 +534,11 @@ def _trim_body(o):
                 try:
                     _clone(".wiki.git")
                 except subprocess.CalledProcessError as e:
-                    if not re.search(_WIKI_ERROR_OK, str(e.output)):
+                    error_ok = False
```

Comment:
Need to check flag after for loop completes. Because err iterates over acceptable errors for different cases, need to check whether `re.search(err, str(e.output)` is true at some iteration.